viewing_party/main.py

```diff
@@ -0,0 +1,129 @@
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched": [movie]
+    }
+    return user_data
```

Comment:
I think this is passing the tests, but this is **replacing** user_data with a new dictionary which has 1 key-value pair with the key watched and the value an array with one movie in it. 

```suggestion
def add_to_watched(user_data, movie):
    user_data["watched"].append(movie)
    return user_data
```